model_compression_toolkit/core/keras/reader/node_builder.py

```diff
@@ -67,12 +67,23 @@ def get_kwargs2index(tfoplambda_layer: TFOpLambda) -> Dict[str, int]:
     Returns:
         A dictionary with argument number and index: {arg_name: arg_index}.
     """
-    if tfoplambda_layer.function in [tf.add, tf.subtract, tf.divide, tf.truediv, tf.multiply, tf.pow,
-                                     tf.matmul, tf.image.crop_and_resize, tf.image.combined_non_max_suppression] or \
-            tfoplambda_layer.symbol in ['__operators__.add', 'math.add', 'math.multiply', 'linalg.matmul', 'concat']:
-        return {arg_name: i for i, arg_name in enumerate(tf_inspect.getfullargspec(tfoplambda_layer.function).args)}
-    else:
-        return {}
+    kwargs2index = {tf.add: {'x': 0, 'y': 1},
```

Comment:
please add some explanation regarding this hardcoded mapping and why we need to work with it like this (I can see a scenario where in a few months we will look at this code and think to ourselves "why did we implement it with this constant dictionary instead of going over the function's argument list).
Also explain the if statements and the difference between the first and the second.
In addition, mention that in order to add support in other functional/positional ops/vars they will need to be added to this mapping.